react-locate-contractor/src/Pages/HomePage.tsx

```diff
@@ -1,13 +1,64 @@
 import './SearchBar.css';
 import TextField from '@mui/material/TextField';
+import { useState } from 'react';
+import { searchBusiness } from '../Services/BusinessService';
+import { business } from './BusinessCard';
+import { SearchCard } from './SearchCard';
+import { Paper, InputBase, IconButton } from '@mui/material';
+import SearchIcon from "@mui/icons-material/Search";
 const HomePage = () => {
+    const [businessData, setBusinessData] = useState<business[]>([])
+    const [searchString, setSearchString] = useState<string>("")
+    const [errorState, setErrorState] = useState<boolean>(false)
+    const search = () => {
+        setErrorState(false)
+        console.log(searchString)
+        searchBusiness(searchString)
+            .then((res) => {
+                const business: business[] = res.data
+                business && setBusinessData(business)
+            })
+            .catch(() => {
+                setErrorState(true)
+            })
+    }
+
     return (
         <div>
             <h1>Home</h1>
-            <TextField placeholder='Search for a business...' className="Search-Bar">
-                <p>SEARCH:</p>
-                <input />
-            </TextField>
+            <Paper
+                component="form"
+                sx={{
+                    p: "2px 4px", display: "flex", alignItems: "center", width: 400,
+                }}
+
+            >
+                <InputBase
+                    onChange={(e) => setSearchString(e.target.value)}
+                    sx={{ ml: 1, flex: 1 }}
+                    placeholder="Search Business"
+
+                    type="text"
+                    onKeyDown={(e) => {
+                        e.key === "Enter" && search()
+                    }}
+
+                />
+                <IconButton onClick={search} sx={{ p: "10px" }} aria-label="search">
+                    <SearchIcon />
+                </IconButton>
+            </Paper>
+            {errorState ? (
+                <>
+                    <p>Error Finding Data For: {searchString}</p>
+                </>
+            ) : (
+                <div className="Beverage-Card-Container">
```

Comment:
Not sure if "Beverage-Card-Container" was an extra class name from the example? You can resolve this comment if you are using it, but you can also remove it since it's extra.